Assets/MRTK/Providers/OpenVR/HPMotionController.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.OpenVR.Input
+{
+    /// <summary>
+    /// Open VR Implementation of the HP Motion Controllers.
+    /// </summary>
+    [MixedRealityController(
+        SupportedControllerType.HPMotionController,
+        new[] { Handedness.Left, Handedness.Right },
+        "Textures/MotionController")]
+    public class HPMotionController : GenericOpenVRController
+    {
+        /// <summary>
+        /// Constructor.
+        /// </summary>
+        public HPMotionController(TrackingState trackingState, Handedness controllerHandedness, IMixedRealityInputSource inputSource = null, MixedRealityInteractionMapping[] interactions = null)
+            : base(trackingState, controllerHandedness, inputSource, interactions)
+        {
+            controllerDefinition = new HPMotionControllerDefinition(inputSource, controllerHandedness);
+        }
+
+        private readonly HPMotionControllerDefinition controllerDefinition;
+
+        /// <inheritdoc />
+        public override float PointerOffsetAngle { get; protected set; } = -30f;
+
+        /// <inheritdoc />
+        public override MixedRealityInteractionMapping[] DefaultLeftHandedInteractions
+        {
+            get
+            {
+                MixedRealityInteractionMapping[] definitionInteractions = controllerDefinition.DefaultLeftHandedInteractions;
+                MixedRealityInteractionMapping[] defaultLeftHandedInteractions = new MixedRealityInteractionMapping[definitionInteractions.Length];
+                for (int i = 0; i < definitionInteractions.Length; i++)
+                {
+                    defaultLeftHandedInteractions[i] = new MixedRealityInteractionMapping(definitionInteractions[i], LeftHandedLegacyInputSupport[i]);
+                }
+                return defaultLeftHandedInteractions;
+            }
+        }
+
+        private static readonly MixedRealityInteractionMappingLegacyInput[] LeftHandedLegacyInputSupport = new[]
+        {
+            new MixedRealityInteractionMappingLegacyInput(), // Spatial Pointer
+            new MixedRealityInteractionMappingLegacyInput(), // Spatial Grip
+            new MixedRealityInteractionMappingLegacyInput(axisCodeX: ControllerMappingLibrary.AXIS_11), // Grip Position
+            new MixedRealityInteractionMappingLegacyInput(axisCodeX: ControllerMappingLibrary.AXIS_11), // Grip Touch
+            new MixedRealityInteractionMappingLegacyInput(axisCodeX: ControllerMappingLibrary.AXIS_11), // Grip Press
```

Comment:
Just caught something here in the OpenVR implementation: to align with the other platforms that "press" == bool, can you update https://github.com/microsoft/MixedRealityToolkit-Unity/blob/1d180dfcacd10332a2f8bfb8cea0f884851ec63e/Assets/MRTK/Core/Providers/UnityInput/GenericJoystickController.cs#L186 to include `|| interactionMapping.InputType == DeviceInputType.GripPress` please?

Reply:
Otherwise, both UWP/Mirage and OpenVR work great! Still need to test out XR SDK with the incoming package